Topics/User_Experience/Color_Theory.md

```diff
@@ -0,0 +1,73 @@
+#### Complementary
+Complementary colors are drawn from two colors on the opposite side of the color wheel. The result is a high-contrast color combination that's bright and that pops, opposing analogous schemes. An example would be red and green. In design, complementary colors are used for emphasis as they focus the attention on a particular object.
+#### Triadic
+Triadic colors are three colors that are equidistant on the color wheel, such that there is no clear superiority of one color. To use a triadic harmony successfully, the colors should be carefully balanced such that one color dominates and the other two are used for accent, adhering to the 60-30-10 rule. In design, triadic schemes are used to curate a lively and cheerful space, suitable for playful or colorful user interfaces.
+
+## Short Analysis of Successful Products
+
+### CTA (Call-To-Action) Buttons
+The goal of call-to-action buttons is to stand out and capture the user's attention. Using the ideas of color harmony and color theory, we know that using complementary colors or bright colors on a white background can help draw the user's attention. In particular, we want to bring out two different kinds of contrasts: 
+* contrast between the button and the background
+* contrast between the button and the text
+
+We also want to pick colors that denote the appropriate feelings that the associated actions represent.
+
+#### Example
+Since creating a repository is something that is incentivized by GitHub, their objective is to draw users' attention to the button as much as possible. GitHub uses the color bright green to show contrast against the otherwise white background, directing users' focus toward the button. The color green also evokes feelings of correctness and positivity, which encourages the users to click on the button.
+<img width="695" alt="Screenshot 2023-11-27 at 10 23 56 AM" src="https://github.com/learning-software-engineering/learning-software-engineering.github.io/assets/107783734/413c350c-3b79-42f1-8360-f3e04848f07b">
+
+On the other hand, deleting a repository is not something that is necessarily encouraged, and can have negative consequences. GitHub portrays this idea by using the color red to evoke feelings of caution and negativity, and to alert the users that this action is dangerous. In addition, the deletion buttons do not stand out from the background as much as the green button (shown above) did, as they had less contrast to the background. This is to prevent drawing the users' attention to these potentially dangerous and consequential actions.
+<img width="633" alt="Screenshot 2023-11-27 at 10 47 32 AM" src="https://github.com/learning-software-engineering/learning-software-engineering.github.io/assets/107783734/6eab108e-b1c1-4d76-8e8a-3c9c61245566">
```

Comment:
I think this a great example! Just a quick question, is there a reason why you chose to analyze github over other products?

Reply:
No specific reason to be honest, just happened to be using GitHub so thought to analyze it at the same time.